hnclient-v2/src/test/java/ca/bc/gov/hlth/hnclientv2/error/ErrorBuilderRouteTest.java

```diff
@@ -8,6 +8,7 @@
 import org.apache.camel.builder.RouteBuilder;
 import org.apache.camel.component.mock.MockEndpoint;
 import org.apache.camel.test.junit5.CamelTestSupport;
+import org.junit.Ignore;
```

Comment:
@Ignore is actually a JUnit 4 annotation and it has been replaced with @Disabled in JUnit 5. This code actually doesn't compile for me...it could be that your IDE is adding the JUnit 4 libraries.

Reply:
The purpose of this test class was to test ErrorProcessor  which is now deprecated so this test class will be removed as a part of HNClient clean-up usery story.